lib/mhv/account_creation/service.rb

```diff
@@ -25,12 +25,16 @@ def create_account(icn:, email:, tou_occurred_at:, break_cache: false)
 
       def create_account_with_cache(icn:, force:, expires_in:, &request)
         cache_hit = true
+        start = nil
         account = Rails.cache.fetch("#{config.service_name}_#{icn}", force:, expires_in:) do
           cache_hit = false
+          start = Time.current
           request.call
         end
-        Rails.logger.info("#{config.logging_prefix} create_account success", { icn:, account:, from_cache: cache_hit })
+        duration_ms = cache_hit == false ? ((Time.current - start) * 1000).round(2) : nil
```

Comment:
if you convert this to a `Duration` object then you can convert to `ms` without exposing this direct math. Also just use `cache_hit`, the `== true/false` part is implied:

`duration_ms = cache_hit ? nil : (Time.current - start).to_s.in_milliseconds`